htdocs/install/page_modcheck.php

```diff
@@ -63,37 +64,51 @@ function xoDiagIfWritable( $path) {
 	return xoDiag( $error ? -1 : 1, $error ? 'Not writable' : 'Writable' );
 }
 
+function imCheckRequirements()
+{
+	$requirement['server_api']['description']=PHP_SAPI;
+	$requirement['server_api']['result']=php_sapi_name();
+	$requirement['server_api']['status']=true;
+
+	$requirement['php_version']['description']=_PHP_VERSION;
+	if (version_compare( phpversion(), '5.6', '>=')) {
+		$requirement['php_version']['status']=1;
+	} else {
+		$requirement['php_version']['status']=0;
+	}
+	$requirement['php_version']['result']=phpversion();
+
+	$requirement['mysql']['description']="MySQL Handler";
+	$requirement['mysql']['result']=in_array("mysql",PDO::getAvailableDrivers(),TRUE) ? "using PDO" : "missing";
+	$requirement['mysql']['status']=in_array("mysql",PDO::getAvailableDrivers(),TRUE) ? true : false;
+
+	$requirement['session']['description']="Session Extension";
+	$requirement['session']['result']=extension_loaded( 'session' ) ? "available" : "missing";
+	$requirement['session']['status']=extension_loaded( 'session' ) ? true : false;
+
+	$requirement['pcre']['description']="PCRE Extension";
+	$requirement['pcre']['result']=extension_loaded( 'PCRE' ) ? "available" : "missing";
+	$requirement['pcre']['status']=extension_loaded( 'PCRE' ) ? true : false;
+
+	$requirement['file_upload']['description']="File uploads";
+	$requirement['file_upload']['result']=xoDiagBoolSetting( 'file_uploads', true ) ? "yes" : "no";
```

Comment:
I think it's better to use _YES and _NO from global language file instead of "yes" and "no"

Reply:
Done

Reply:
Done